dockerfile/anaconda-ci/requirements.txt

```diff
@@ -0,0 +1,21 @@
+# pip package install list
+# storing it as an independent file enables Dependabot to bump versions with noise and attention
+# silent updates of pylint and astroid usually cause breakages
+# for format details see https://pip.pypa.io/en/stable/reference/requirements-file-format/
+
+# unit tests of all kinds
+pocketlint
```

Comment:
We don't use specific versions, so I wonder what will Dependabot do with that.

Reply:
Oh, I assumed that dependabot will just always resolve that to latest and be quiet. But that's actually a good question.

Reply:
I can't find any mentions of this, let alone documentation. Taking a glance at the code, I *think* packages with no version are just skipped over. We can try, and revert if it's not working.